lib/widget/button.dart

```diff
@@ -99,9 +99,13 @@ class ButtonState extends WidgetState<Button> {
     }
 
     if(widget._controller.borderGradient!=null){
-      return CustomPaint(
-        foregroundPainter: _Painter(widget._controller.borderGradient!, widget._controller.borderRadius?.getValue(), widget._controller.borderWidth?.toDouble()??Size.zero.width) ,
-        child: rtn,
+      return Padding(
```

Comment:
I understand it's minor but don't add a UI widget unless you have to. Write more logic if you have to, but minimize the wastefulness of wrapping widgets inside widgets if you don't have to.
```
if (has border gradient)
  rtn = CustomPaint()

if (has padding)
  rtn = Padding(child: rtn);
```